com.dec05eba.git.gpu_screen_recorder.yml

```diff
@@ -0,0 +1,79 @@
+app-id: com.dec05eba.git.gpu_screen_recorder
```

Comment:
I don't think there should be `git` in the app-id.

Reply:
fixed